data/src/main/java/com/microsoft/azure/kusto/data/Utils.java

```diff
@@ -67,21 +67,27 @@ public static String getPackageVersion() {
     }
 
     public static String formatDurationAsTimespan(Duration duration) {
-        long seconds = duration.getSeconds();
+        long durationInSeconds = duration.getSeconds();
         int nanos = duration.getNano();
-        long hours = TimeUnit.SECONDS.toHours(seconds) % TimeUnit.DAYS.toHours(1);
-        long minutes = TimeUnit.SECONDS.toMinutes(seconds) % TimeUnit.MINUTES.toSeconds(1);
-        long secs = seconds % TimeUnit.MINUTES.toSeconds(1);
-        long days = TimeUnit.SECONDS.toDays(seconds);
-        String positive = String.format(
-                "%02d.%02d:%02d:%02d.%.3s",
```

Comment:
My understanding is that the `timespan` standard does not include the days if there are none, nor the subseconds if there are none.